### DISCUSSION TITLE
Quick cart: cart icon selection

### DISCUSSION THREAD

**Discussion by satyajittalukder** [getdokan/storegrowth-sales-booster#426] [Enhancement]
![image](https://github.com/Invizo/storegrowth-sales-booster/assets/65698588/a25f890d-7470-4583-beff-f5f3cc451d06)

the cart icon selection option should be like this.

**Comment by satyajittalukder** [comment]
@hz-tyfoon

**Comment by imtiaz-pranto** [comment]
Converting it into discussion for further processing
